spec/models/user_spec.rb

```diff
@@ -44,7 +44,73 @@
 
     expect(user.items.count).to eq(3)
     expect(user.open_items.count).to eq(2)
+  end
+
+  it "cannot find won auctions if the user has no bidders" do
+      user = create(:user)
+      expect(user.bids).to be_blank
+      expect(user.won_items.count).to eq(0)
+  end
+
+  it "can identify a single auction won by a specific user" do
+    user = create(:user_with_bids)
+
+    item1, item2, item3 = user.items
 
+    dup_bid = create(:bid, price: 1000, item: item1, user: user)
+    item1.update(status: "closed")
+
+    expect(user.won_items.count).to eq(1)
+    expect(user.open_items.count).to eq(2)
+    expect(user.closed_items.count).to eq(1)
   end
+    
+  it "can find a multiple auctions won by a specific user" do
+    user = create(:user_with_bids)
 
+    item1, item2, item3 = user.items
+
+    item1.update(status: "closed")
+    item2.update(status: "closed")
+
+    expect(user.won_items.count).to eq(2)
+    expect(user.open_items.count).to eq(1)
+    expect(user.closed_items.count).to eq(2)
+  end
+
+  it "can find multiple auctions lost by a specific user" do
+
+    user1 = create(:user_with_bids)
+
+    item1, item2, item3 = user1.items
+
+    dup_bid = create(:bid, price: 100, item: item1, user: user1)
```

Comment:
Same thing here.  Adding duplicate bid to make sure duplicate  items don't show up in the dashboard.